src/components/Atoms/FormInput/FormInput.module.css

```diff
@@ -0,0 +1,124 @@
+.FormInput {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 8px;
+
+  background-color: none;
+
+  border-style: solid;
+  border-radius: 5px;
+  border-width: 1px;
+
+  max-width: 100%;
+  max-height: 100%;
+
+  padding: 13px;
+
+  cursor: text;
+}
+
+.FormInput div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex: 1 1 1;
+}
+
+.FormInputText {
+  font-size: 0.875rem;
+  font-weight: 400;
+  opacity: 1;
+
+  background: none;
+  border: 0;
+  outline: 0;
+
+
+  min-width: 200px;
+  flex: 1 0 1;
+}
+
+.FormInputText::placeholder {
+  opacity: 1;
+}
+
+.FormIcon {
+  width: 18px;
+  height: 18px;
+}
+
+.dark *,
+.FormInput:has(.dark) {
+  border-color: var(--popularDescriptionText);
+  color: var(--popularDescriptionText);
+  stroke: var(--popularDescriptionText);
+  transition: ease-in-out 0.25s;
+}
+
+.dark:hover *,
+.FormInput:has(.dark):hover {
+  border-color: var(--primaryColor);
+  color: var(--primaryColor);
+  stroke: var(--primaryColor);
+  transition: ease-in-out 0.25s;
+}
+
+.light *,
+.light path {
+  border-color: var(--userEditDark);
+  color: var(--userEditDark);
+  stroke: var(--userEditDark);
+  transition: ease-in-out 0.25s;
```

Comment:
we got transition only on icon ❌ 
implement it on `div/label` as well 💡

Reply:
What do you mean? To me it looks like a transition on the entire element 🤔
![obraz](https://github.com/KaczorowskaKarolina/soyummy_frontend/assets/124502546/6ebb0ab4-b064-4075-acb9-a237c26ee5a4)

Reply:
@dag-szad now it looks like label and placeholder shine and few secinds later icon starts shine, it doesn't work together at the same time. Please change this 🙏